onnx/checker.cc

```diff
@@ -304,6 +304,131 @@ void check_attribute(
   }
 }
 
+void verify_node_schema(
+    const NodeProto& node,
+    const OpSchema* schema,
+    int domain_version) {
+  if (!schema || schema->Deprecated()) {
+    // There's no primitive operator for the node.
+    // Check whether it's referring to a function.
```

Comment:
Is this an out-dated comment? I don't understand it.

Reply:
It is indeed a outdated comment. Removed